lib/leagueable.rb

```diff
@@ -1,11 +1,10 @@
 require 'pry'
 
 module Leagueable
-  # Have to adjust ALL methods and helper methods in Leagueable to accept and recognize team_id as a string.
```

Comment:
This seems like an important thing. Didn't Ian tell us to convert the team ids? Maybe I am mistaken but we can correct that when need be.

Reply:
Note: I  added a string to int converter to my iteration_4 methods for now. It just converts the method input to an int just to make sure our methods pass the tests.

Reply:
I will keep an eye out for this as I work through iteration 4 this morning. I don't think it will be too big of a deal to adjust as we get there.